dashboard/public/javascripts/generateLineChart.js

```diff
@@ -17,7 +17,7 @@ function generateLineChart(id, tdID, title) {
     var layout = {
     xaxis: {range: [0, xmax]},
     title: "",
-    width: 220,
+    width: 300,
```

Comment:
I tried to make it use auto sizing with `automargin: true,` for both x and y axis, as well as with using `autoscale: true`, but both made the plot way to small. We might need to experiment with this a bit more rather than using fixed width. Similar to what I need to do for the rest of the buttons and such, tables should be doing that already though.